redis/src/main/scala/zio/redis/RedisClusterConfig.scala

```diff
@@ -0,0 +1,18 @@
+package zio.redis
+
+import zio.Chunk
+
+final case class RedisUri(host: String, port: Int) {
+  override def toString: String = s"$host:$port"
+}
+
```

Comment:
Please keep a single top-level artifact per file. Also, please revert renaming `redis.scala` for the same reason.

Reply:
>Also, please revert renaming redis.scala for the same reason.

Sry, I don't understand what i should revert

Reply:
There was a file named redis.scala which you renamed to Redis.scala.

Reply:
Oh. Ok, done. I thought it was spelling mistake )